src/lib/operators/jit_operator/jit_operations.cpp

```diff
@@ -84,14 +127,20 @@ bool jit_not_like(const std::string& a, const std::string& b) {
   return !std::regex_match(a, regex);
 }
 
-void jit_is_null(const JitTupleValue& lhs, const JitTupleValue& result, JitRuntimeContext& context) {
-  result.set_is_null(false, context);
-  result.set<bool>(lhs.is_null(context), context);
+JitValue<bool> jit_is_null(const JitExpression& left_side, JitRuntimeContext& context) {
+  switch (left_side.result().data_type()) {
+    BOOST_PP_SEQ_FOR_EACH_PRODUCT(JIT_IS_NULL_CASE, (JIT_DATA_TYPE_INFO))
```

Comment:
Could you add a comment on why you need this <s>ugly</s> beautiful macro-based meta programming?